app/controllers/api/v1/referees_controller.rb

```diff
@@ -30,7 +31,12 @@ def update
         update_national_governing_bodies(national_governing_body_ids) if national_governing_body_ids.present?
 
         if @referee.update!(permitted_params)
-          json_string = RefereeSerializer.new(@referee).serialized_json
+          serializer_options = {
```

Comment:
since this is the excat same variable in two places let's pull this out to a private helper method, something like: 
```
private

def serializer_options
  @serializer_options ||= {
    include: %i[national_governing_bodies certifications],
    params: { current_user: current_referee }
  }
end
```

when you declare an instance variable in ruby like `@all_referees ||= Referee.all` it "memoizes" that variable so calling it multiple times doesn't calculate the same value over and over.